edgexfoundry/app-service-configurable
# refactor!: Use latest SDK for flattened config stem

BREAKING CHANGE: Location of service configuration in Consul changed

Also required updating for v3 modules

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
Run no-secure EdgeX stack
Build and run from command line 
```
./app-service-configurable -p=sample -cp
```
Verify  service's config is in Consul at `edgex/v3/app-sample`

## New Dependency Instructions (If applicable)
